changelog.md

```diff
@@ -126,6 +126,7 @@
   Added `HoleyEnum` for enums with holes, `OrdinalEnum` for enums without holes.
   Added `hasClosure`.
   Added `pointerBase` to return `T` for `ref T | ptr T`.
+  Added `toSigned`, `toUnsigned`.
```

Comment:
Where is the RFC? Who asked for these API extensions? We cannot simply add everything that is occasionally useful...